service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingBookingService.java

```diff
@@ -240,9 +158,4 @@ private boolean isPastHearing(Element<HearingBooking> element) {
             .build();
     }
 
-    private String buildNewHearingLabel(HearingBooking hearingBooking, int i) {
```

Comment:
what about other not used methods?

removePastHearings, getPastHearings, getMostUrgentHearingBooking, combineHearingDetails, setHearingJudge, resetHearingJudge, getNewHearings